packages/cli/openapi-parser/src/v3/extensions/fernExtensions.ts

```diff
@@ -232,7 +232,27 @@ export const FernOpenAPIExtension = {
      *          name: Console Log My Endpoint
      *          description: This is a code sample that logs the response
      */
-    EXAMPLES: "x-fern-examples"
+    EXAMPLES: "x-fern-examples",
+
+    /**
+     * securitySchemes:
+     *   Bearer:
+     *     scheme: http
+     *     type: bearer
+     *     x-fern-auth-environment-variable-name: MY_AUTH_TOKEN
+     */
+    AUTH_ENVIRONMENT_VARIABLE_NAME: "x-fern-auth-environment-variable-name",
+    /**
+     * For basic auth, you must specify the environment variable for both fie;lds
+     * securitySchemes:
+     *   Basic:
+     *     scheme: http
+     *     type: basic
+     *     x-fern-auth-username-environment-variable-name: BASIC_USERNAME
+     *     x-fern-auth-password-environment-variable-name: BASIC_PASSWORD
+     */
+    AUTH_USERNAME_ENVIRONMENT_VARIABLE_NAME: "x-fern-auth-username-environment-variable-name",
+    AUTH_PASSWORD_ENVIRONMENT_VARIABLE_NAME: "x-fern-auth-password-environment-variable-name"
```

Comment:
these are going to get pretty gross. thoughts on changing the format to 

```
x-fern-bearer: 
  name: apiKey # replaces `x-fern-bearer-token-variable-name` 
  env: MY_AUTH_TOKEN
```
and then same for basic 

```
x-fern-basic: 
  username: 
    name: apiKey # replaces `x-fern-username-variable-name` 
    env: MY_AUTH_TOKEN
  password: 
    ...
```

Reply:
hahaha yeah the naming was rough, this seems reasonable
i dont love that this is essentially duplicating the openapi they already have though

Reply:
ah nevermind,  i see your replaces comments! yeah cool with this

Reply:
we'll still need to support the old ones for backcompat but will effectively replace